### ISSUE TITLE
UnicodeDecodeError: 'ascii' codec can't decode byte 0xb3 in position 13: ordinal not in range(128)

### ISSUE THREAD

**Issue by ltfdick1994** [feross/SpoofMAC#109]
I changed it to utf-8 but still don't work.
For ascii it shows "Ordinal Not In Range"
How to get this error fixed?
I am totally stuck here.
Thanks for your attention!

**Comment by h-enes-simsek** [comment]
changed return result.decode('ascii') to return result.decode('utf-8') in interface.py and it works.

**Comment by diegomustafamarino** [comment]
> changed return result.decode('ascii') to return result.decode('utf-8') in interface.py and it works.

No, it's not working on Windows 10. :/ I have the same problem.

**Comment by diegomustafamarino** [comment]
> I changed it to utf-8 but still don't work. For ascii it shows "Ordinal Not In Range" How to get this error fixed? I am totally stuck here. Thanks for your attention!

Did you solve?

**Comment by 0w0-Yui** [comment]
> > I changed it to utf-8 but still don't work. For ascii it shows "Ordinal Not In Range" How to get this error fixed? I am totally stuck here. Thanks for your attention!
> 
> Did you solve?

return result.decode('latin-1') works fine with me, win10 21h2
